tests/archetypes/test_gambler.py

```diff
@@ -100,3 +130,46 @@ def test_repr(self):
 
         self.assertEqual(gambler_params.__repr__(),
                          '1:1:1:0.05|0.07|0.7|0.03|0.6|0.7|0.45|0.1')
+
+    def test_crossover(self):
+        plays=1
```

Comment:
nit: spacing around `=`

I like what you did above with `plays, op_plays, op_start_plays = 1, 1, 1`